frontend/Components/Header.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import {
+	Link,
+} from 'react-router-dom';
+export interface HeaderParams{
+	pageName: string;
+}
+export class MainHeader extends React.Component<HeaderParams> {
```

Comment:
Rename it to `Header` or the file to `MainHeader` - the file name should match the primary export of the file (if there is one)